### ISSUE TITLE
Possibility of Handoff on MacBook Air Late 2010 (AirPort Card: BCM94360CS2)

### ISSUE THREAD

**Issue by joshuapurwadi** [dokterdok/Continuity-Activation-Tool#41]
Hi,

I'm wondering if the upgrade to this card (BCM94360CS2) on MBA Late 2010 is still possible.
I believe that I saw MBA Late 2010 in the table in the main page which says Handoff is possible with card upgrade a few days ago but it's magically gone from the table when I checked it just now.

The entry is still there in this thread regarding MBA Late 2010:
http://forums.macrumors.com/showthread.php?p=20124161#post20124161

I'm aware that minor tweaks will be required as the card (BCM94360CS2) sized up a bit longer than the original card and the antenna cable will need to be clipped off from the screws' bracket, but I don't really mind as long as Handoff is working with BCM94360CS2.

I hope someone can elaborate in this matter as me myself have already ordered the card and am waiting for it to arrive. I hope that my purchase won't go to waste.

Regards,
Josh..

**Comment by dnivi3** [comment]
@joshuapurwadi: how did you manage to get the small cable connectors to reach all the way to the end of the BCM94360CS2's connectors? My small cable connectors connecting to the WiFi-card seem very non-stretchable.

**Comment by philmorse** [comment]
In fact, I didn't.  I have seen where others disconnected the cable from the hinge to get more play and were able to re-route.  In my case, the connectors broke when I was removing them from the old bluetooth card so I bought an antenna on Ebay and soldered the new connectors.  Results have been poor because the wires are now exposed to interference.   I tried shielding with foil and then copper tape and things have just gotten worse.  I intend to have the antenna replaced altogether (beyond my skills), go back to square one, and stop messing around.  It's not worth the trouble.

**Comment by dnivi3** [comment]
Can confirm that I managed to install the BCM94360CS2 in my 2010 MBA '13 after following instructions and pictures in [a previous comment](https://github.com/dokterdok/Continuity-Activation-Tool/issues/41#issuecomment-64079871).

**Comment by tikomile** [comment]
thanks ,  handoff working for me in  my MBA 13' !! after resign icloud account !!!!

**Comment by hansol2005** [comment]
After reading this thread, i've bought a BCM94360CS2 from eBay. for my 2010MBA but the only thing is that no where in this thread does it say how to secure the card as the screw won't fit. Or do you just leave it not screwed in??

**Comment by joshuapurwadi** [comment]
Just leave it unscrewed, it won’t detach itself.

**Comment by tikomile** [comment]
Hansol Make sure the insulation on the motherboard !!!

**Comment by hansol2005** [comment]
Thanks for quick reply,

**Comment by hansol2005** [comment]
I got continuity working with installing a BCM94360CS2 in my 2010 MBA but can't get instant hotspot to work..
![screen shot 2015-03-18 at 16 57 58](https://cloud.githubusercontent.com/assets/11265475/6714503/8081f4c4-cd90-11e4-8100-ecb5f1f9de11.png)
My Mac sees it but won't connect ..
![screen shot 2015-03-18 at 16 57 36](https://cloud.githubusercontent.com/assets/11265475/6714516/9920f656-cd90-11e4-8d92-e5d96303a1ea.png)
can someone please help..

**Comment by tikomile** [comment]
u have 10.10.2 ?

**Comment by hansol2005** [comment]
Yes I've 10.10.2. I've Tried resetting network settings on iPhone, logging out of iCloud on iPhone.  Changing name of iPhone. Rebooting mba but nothing works.

**Comment by tikomile** [comment]
but handoff is working , yes ?

**Comment by hansol2005** [comment]
Yes

**Comment by tikomile** [comment]
can u connect to the hotspot with other devices ?

**Comment by hansol2005** [comment]
I can connect to the hotspot if I switch it in first in iPhone settings, then switch wifi off then on on my mba.

**Comment by tikomile** [comment]
i think, this problem with iphone six, cz i have i6 too and i cant connect to any device in my home and office : iOS 8.2 i think

**Comment by hansol2005** [comment]
Thanks for your help.

**Comment by pengusel** [comment]
hi, i use macbook air 2011 os x 10.11 el capitan,  if i change and use BCM943224PCIEBT2BX no need patch? pls help.

**Comment by tikomile** [comment]
With OS X El Capitan , you need to use patch with the BCM94360CS2 , just open it w right click / open. and use activate method ! all work for me :)

**Comment by csundar** [comment]
so MBA 13" late 2010 with El Capitan.  I have upgraded to the BCM94360CS2 and see BT LE, Handoff support, and airdrop all listed as working now per system profiler (just like swill00net's images).  I do not see instant hotspot and continuity is not working either.  Any advice? I am able to use airdrop now.

**Comment by tikomile** [comment]
csundar, just use Continuity-Activation-Tool, and follow the instruction ! with el capitan last update only BCM94360CS2 adapter change not working anymore :

**Comment by csundar** [comment]
So ran through the CAT tool and nothing changed.  Same results from my setup.  Output of diagnostics.

--- Initiating system compatiblity check ---

--- Hardware/OS checks ---
Verifying Continuity status...          OK. OS X reports Continuity as active
Verifying Mac model reference...        OK. Known compatible Mac Model detected: MacBookAir3,2
Verifying Mac board-id...               OK. Long board id detected: Mac-942C5DF58193131B
Verifying OS X version...               Warning: This version of Mac OS X (10.11.2) is Experimental! Only partially tested on El Capitan
Verifying Wi-Fi hardware...             OK. A Broadcom AirPort card is active, and is using the Continuity compatible Brcm4360 kext
Verifying AWDL status...                OK. An AWDL interface is up, Wi-Fi is ready for Continuity
Verifying Bluetooth hardware...         OK. The internal Bluetooth card is active
Verifying Bluetooth version...          OK. Bluetooth 4.0 detected
Verifying Bluetooth features...         OK. Bluetooth features are Continuity compliant
Verifying Bluetooth firmware...         OK. Bluetooth firmware version: v112 c9037
Verifying Login Item...                 OK. Login item for Auto Continuity Check is not set.

--- Modifications check ---
Verifying OS kext protection...         OK. Kext developer mode is not active. This tool can fix this.
Verifying SIP...                        Ok. System Integrity Protection is already disabled
Verifying ContinuitySupport...          OK. Already patched.
Verifying kexts readability...          OK. Wi-Fi and Bluetooth kexts were found and could be read
Verifying Wi-Fi whitelist status...     OK. Your board-id is not yet whitelisted. This tool can fix this.
Verifying BT4 dongles compatibility...  OK. Compatibility with BT4 USB dongles is not enabled, this tool can fix this if a dongle is plugged in
Verifying old Wi-Fi kext presence...    OK. Legacy Brcm4331 Wi-Fi driver is present. This tool can fix this.
Verifying legacy Wi-Fi card patch...    OK. The legacy Wi-Fi patch is not present. This tool can fix this.
--- Modifications check ---

Ideas?

**Comment by tikomile** [comment]
try sign out/in iCloud settings after patching

**Comment by csundar** [comment]
done that twice with no change

**Comment by tikomile** [comment]
try to clean install osx with usb drive

**Comment by csundar** [comment]
so looks like it's my phone.  as I was sitting here backing things up, handoff popped up...  working on that next.  

thanks for the help

**Comment by asoksevil** [comment]
Did any of you experience a weak WiFi signal or reduce range? Since I have installed this WiFi spots that I was previously able to connect have reduced drastically. Before the new chip, airport would show 5 different wifi spots but at this moment I can't see any of them.

**Comment by diggity-byte** [comment]
Upgrading to BT4.0 and 802.11ac appears to be more or less plug and play...
![MBA2010_wireless_upgrade](https://user-images.githubusercontent.com/76860465/103451364-1cecce80-4c78-11eb-9f84-8c54b2400898.png)
